packages/app/prisma-utils/test/DbCleaner.ts

```diff
@@ -5,12 +5,12 @@ export enum DB_MODEL {
 	item2 = 'item2',
 }
 
-export async function cleanTables(prisma: PrismaClient, modelNames: readonly DB_MODEL[]) {
-	const delegates = modelNames.map<{ deleteMany: () => Promise<unknown> }>(
-		(modelName) => prisma[modelName],
-	)
-
-	for (const delegate of delegates) {
-		await delegate.deleteMany()
+export async function cleanTables(
+	prisma: PrismaClient,
+	modelNames: readonly DB_MODEL[],
```

Comment:
@kibertoad Technically we can make `DB_MODEL` generic and use function from lib instead of copying it every time, WDYT?

Reply:
yup, that would be great